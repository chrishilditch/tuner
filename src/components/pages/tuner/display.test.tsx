import { expect, test } from "vitest";
import { render } from "vitest-browser-react";
import Display from "./display";

test("basic render test", async () => {
  const { getByText } = render(
    <Display pitch={82} activeNote={{ name: "E", frequency: 82.41 }} />
  );

  await expect.element(getByText("E")).toBeInTheDocument();
  await expect.element(getByText("82.41")).toBeInTheDocument();
});

test("E tuning - match", async () => {
  const { getByTestId } = render(
    <Display pitch={82} activeNote={{ name: "E", frequency: 82.41 }} />
  );

  await expect.element(getByTestId("active-0")).toBeInTheDocument();
  await expect.element(getByTestId("inactive--1")).toBeInTheDocument();
  await expect.element(getByTestId("inactive-1")).toBeInTheDocument();
});

test("E tuning - one under", async () => {
  const { getByTestId } = render(
    <Display pitch={81} activeNote={{ name: "E", frequency: 82.41 }} />
  );

  await expect.element(getByTestId("inactive-0")).toBeInTheDocument();
  await expect.element(getByTestId("active--1")).toBeInTheDocument();
  await expect.element(getByTestId("inactive-1")).toBeInTheDocument();
});

test("E tuning - one over", async () => {
  const { getByTestId } = render(
    <Display pitch={83.5} activeNote={{ name: "E", frequency: 82.41 }} />
  );

  await expect.element(getByTestId("inactive-0")).toBeInTheDocument();
  await expect.element(getByTestId("inactive--1")).toBeInTheDocument();
  await expect.element(getByTestId("active-1")).toBeInTheDocument();
});

test("E tuning - four over", async () => {
  const { getByTestId } = render(
    <Display pitch={86.5} activeNote={{ name: "E", frequency: 82.41 }} />
  );

  await expect.element(getByTestId("inactive-3")).toBeInTheDocument();
  await expect.element(getByTestId("active-4")).toBeInTheDocument();
  await expect.element(getByTestId("inactive-5")).toBeInTheDocument();
});

test("E tuning - 10 over", async () => {
  const { getByTestId } = render(
    <Display pitch={92.42} activeNote={{ name: "E", frequency: 82.41 }} />
  );

  await expect.element(getByTestId("inactive-9")).toBeInTheDocument();
  await expect.element(getByTestId("active-10")).toBeInTheDocument();
  await expect.element(getByTestId("inactive-11")).toBeInTheDocument();
});

test("E tuning - lots over", async () => {
  const { getByTestId } = render(
    <Display pitch={100} activeNote={{ name: "E", frequency: 82.41 }} />
  );

  await expect.element(getByTestId("inactive-10")).toBeInTheDocument();
  await expect.element(getByTestId("active-11")).toBeInTheDocument();
});

test("E tuning - match - snapshot", async () => {
  const { container } = render(
    <Display pitch={82} activeNote={{ name: "E", frequency: 82.41 }} />
  );
  expect(container.innerHTML).toMatchSnapshot();
});

test("E tuning - three over - snapshot", async () => {
  const { container } = render(
    <Display pitch={85} activeNote={{ name: "E", frequency: 82.41 }} />
  );
  expect(container.innerHTML).toMatchSnapshot();
});

test("E tuning - lots over - snapshot", async () => {
  const { container } = render(
    <Display pitch={100} activeNote={{ name: "E", frequency: 82.41 }} />
  );
  expect(container.innerHTML).toMatchSnapshot();
});
