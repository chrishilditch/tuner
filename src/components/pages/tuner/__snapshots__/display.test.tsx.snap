// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`E tuning - lots over - snapshot 1`] = `"<div class="flex flex-col items-center"><svg width="100" height="100" class="w-100 h-70 max-w-full" viewBox="-50 -50 100 70"><path d="M2.610523844401034,-19.828897227476208,4.838437911993354,-19.40591452551993 L10.886485301985045,-43.66330768241984,5.8736786499023275,-44.61501876182147 Z" fill="#00d8ca" stroke="#004b59" stroke-width="0.1" class="transition-fill duration-100 delay-75" data-testid="inactive-1"></path><path d="M5.512747116339983,-19.225233918766378,7.653668647301797,-18.477590650225736 L17.22075445642904,-41.5745789630079,12.403681011764963,-43.25677631722435 Z" fill="#00d8ca" stroke="#004b59" stroke-width="0.1" class="transition-fill duration-100 delay-75" data-testid="inactive-2"></path><path d="M8.293864853124782,-18.199225417530865,10.300761498201084,-17.143346014042248 L23.176713370952438,-38.572528531595054,18.661195919530762,-40.94825718944445 Z" fill="#00d8ca" stroke="#004b59" stroke-width="0.1" class="transition-fill duration-100 delay-75" data-testid="inactive-3"></path><path d="M10.892780700300541,-16.773411358908483,12.721564405555279,-15.4324916677544 L28.62351991249938,-34.7231062524474,24.50875657567622,-37.740175557544084 Z" fill="#00d8ca" stroke="#004b59" stroke-width="0.1" class="transition-fill duration-100 delay-75" data-testid="inactive-4"></path><path d="M13.25240096431475,-14.979114415780042,14.862896509547884,-13.382612127177165 L33.44151714648274,-30.11087728614862,29.817902169708187,-33.70300743550509 Z" fill="#00d8ca" stroke="#004b59" stroke-width="0.1" class="transition-fill duration-100 delay-75" data-testid="inactive-5"></path><path d="M15.32088886237956,-12.855752193730785,16.677716441343364,-11.038739706241163 L37.52486199302257,-24.837164339042616,34.47199994035401,-28.925442435894265 Z" fill="#00d8ca" stroke="#004b59" stroke-width="0.1" class="transition-fill duration-100 delay-75" data-testid="inactive-6"></path><path d="M17.052803287081844,-10.449971294318976,18.126155740732997,-8.452365234813989 L40.783850416649244,-19.017821778331474,38.36880739593415,-23.512435412217695 Z" fill="#00d8ca" stroke="#004b59" stroke-width="0.1" class="transition-fill duration-100 delay-75" data-testid="inactive-7"></path><path d="M18.410097069048806,-7.814622569785476,19.17639469736386,-5.680306894078453 L43.14688806906869,-12.78069051167652,41.42271840535982,-17.58290078201732 Z" fill="#00d8ca" stroke="#004b59" stroke-width="0.1" class="transition-fill duration-100 delay-75" data-testid="inactive-8"></path><path d="M19.362952807562156,-5.007600081088829,19.805361374831406,-2.7834620192013086 L44.56206309337067,-6.262789543202945,43.56664381701485,-11.267100182449864 Z" fill="#00d8ca" stroke="#004b59" stroke-width="0.1" class="transition-fill duration-100 delay-75" data-testid="inactive-9"></path><path d="M19.890437907365467,-2.0905692653530696,19.999238461283426,0.1745307099674787 L44.99828653788771,0.39269409742682704,44.7534852915723,-4.703780847044406 Z" fill="#00d8ca" stroke="#004b59" stroke-width="0.1" class="transition-fill duration-100 delay-75" data-testid="inactive-10"></path><path d="M19.48144032084623,0.850578053624052,19.259922641605186,3.050472068284502 L45.43366366737634,7.19598539185062,45.95621819276546,2.006491818805456 Z" fill="#004b59" stroke="#004b59" stroke-width="0.1" class="transition-fill duration-100 delay-75" data-testid="active-11"></path><path d="M-1.5688033694578483,-17.93150456565142,1.5688033694578465,-17.93150456565142 L4.0963199091399325,-46.82115081031204,-4.096319909139937,-46.82115081031204 Z" fill="#00d8ca" stroke="#004b59" stroke-width="0.1" class="transition-fill duration-100 delay-75" data-testid="inactive-0"></path><path d="M-2.6105238444010315,-19.828897227476208,-4.8384379119933545,-19.40591452551993 L-10.886485301985047,-43.66330768241984,-5.87367864990232,-44.61501876182147 Z" fill="#00d8ca" stroke="#004b59" stroke-width="0.1" class="transition-fill duration-100 delay-75" data-testid="inactive--1"></path><path d="M-5.512747116339984,-19.225233918766378,-7.653668647301798,-18.477590650225736 L-17.220754456429045,-41.5745789630079,-12.403681011764965,-43.25677631722435 Z" fill="#00d8ca" stroke="#004b59" stroke-width="0.1" class="transition-fill duration-100 delay-75" data-testid="inactive--2"></path><path d="M-8.293864853124784,-18.19922541753086,-10.300761498201087,-17.143346014042244 L-23.17671337095245,-38.57252853159505,-18.661195919530762,-40.94825718944444 Z" fill="#00d8ca" stroke="#004b59" stroke-width="0.1" class="transition-fill duration-100 delay-75" data-testid="inactive--3"></path><path d="M-10.892780700300547,-16.77341135890848,-12.721564405555277,-15.432491667754402 L-28.623519912499372,-34.72310625244741,-24.50875657567623,-37.74017555754408 Z" fill="#00d8ca" stroke="#004b59" stroke-width="0.1" class="transition-fill duration-100 delay-75" data-testid="inactive--4"></path><path d="M-13.252400964314749,-14.979114415780044,-14.862896509547884,-13.382612127177165 L-33.44151714648274,-30.11087728614862,-29.817902169708184,-33.7030074355051 Z" fill="#00d8ca" stroke="#004b59" stroke-width="0.1" class="transition-fill duration-100 delay-75" data-testid="inactive--5"></path><path d="M-15.32088886237956,-12.855752193730787,-16.677716441343364,-11.038739706241163 L-37.52486199302257,-24.837164339042616,-34.47199994035401,-28.925442435894272 Z" fill="#00d8ca" stroke="#004b59" stroke-width="0.1" class="transition-fill duration-100 delay-75" data-testid="inactive--6"></path><path d="M-17.052803287081847,-10.449971294318976,-18.126155740733,-8.452365234813987 L-40.78385041664925,-19.01782177833147,-38.368807395934155,-23.512435412217695 Z" fill="#00d8ca" stroke="#004b59" stroke-width="0.1" class="transition-fill duration-100 delay-75" data-testid="inactive--7"></path><path d="M-18.410097069048806,-7.814622569785476,-19.17639469736386,-5.680306894078453 L-43.14688806906869,-12.78069051167652,-41.42271840535981,-17.58290078201732 Z" fill="#00d8ca" stroke="#004b59" stroke-width="0.1" class="transition-fill duration-100 delay-75" data-testid="inactive--8"></path><path d="M-19.362952807562156,-5.007600081088828,-19.805361374831406,-2.7834620192013073 L-44.56206309337067,-6.262789543202941,-43.56664381701485,-11.267100182449862 Z" fill="#00d8ca" stroke="#004b59" stroke-width="0.1" class="transition-fill duration-100 delay-75" data-testid="inactive--9"></path><path d="M-19.890437907365467,-2.090569265353071,-19.999238461283426,0.17453070996747794 L-44.99828653788771,0.3926940974268253,-44.7534852915723,-4.70378084704441 Z" fill="#00d8ca" stroke="#004b59" stroke-width="0.1" class="transition-fill duration-100 delay-75" data-testid="inactive--10"></path><path d="M-19.48144032084623,0.8505780536240521,-19.259922641605186,3.050472068284503 L-45.43366366737634,7.1959853918506225,-45.95621819276546,2.0064918188054564 Z" fill="#00d8ca" stroke="#004b59" stroke-width="0.1" class="transition-fill duration-100 delay-75" data-testid="inactive--11"></path><text x="0" y="4" text-anchor="middle" alignment-baseline="middle" class="text-4xl">E</text><text x="0" y="17" text-anchor="middle" alignment-baseline="middle" class="text-[4px]">82.41</text></svg></div>"`;

exports[`E tuning - match - snapshot 1`] = `"<div class="flex flex-col items-center"><svg width="100" height="100" class="w-100 h-70 max-w-full" viewBox="-50 -50 100 70"><path d="M2.610523844401034,-19.828897227476208,4.838437911993354,-19.40591452551993 L10.886485301985045,-43.66330768241984,5.8736786499023275,-44.61501876182147 Z" fill="#00d8ca" stroke="#004b59" stroke-width="0.1" class="transition-fill duration-100 delay-75" data-testid="inactive-1"></path><path d="M5.512747116339983,-19.225233918766378,7.653668647301797,-18.477590650225736 L17.22075445642904,-41.5745789630079,12.403681011764963,-43.25677631722435 Z" fill="#00d8ca" stroke="#004b59" stroke-width="0.1" class="transition-fill duration-100 delay-75" data-testid="inactive-2"></path><path d="M8.293864853124782,-18.199225417530865,10.300761498201084,-17.143346014042248 L23.176713370952438,-38.572528531595054,18.661195919530762,-40.94825718944445 Z" fill="#00d8ca" stroke="#004b59" stroke-width="0.1" class="transition-fill duration-100 delay-75" data-testid="inactive-3"></path><path d="M10.892780700300541,-16.773411358908483,12.721564405555279,-15.4324916677544 L28.62351991249938,-34.7231062524474,24.50875657567622,-37.740175557544084 Z" fill="#00d8ca" stroke="#004b59" stroke-width="0.1" class="transition-fill duration-100 delay-75" data-testid="inactive-4"></path><path d="M13.25240096431475,-14.979114415780042,14.862896509547884,-13.382612127177165 L33.44151714648274,-30.11087728614862,29.817902169708187,-33.70300743550509 Z" fill="#00d8ca" stroke="#004b59" stroke-width="0.1" class="transition-fill duration-100 delay-75" data-testid="inactive-5"></path><path d="M15.32088886237956,-12.855752193730785,16.677716441343364,-11.038739706241163 L37.52486199302257,-24.837164339042616,34.47199994035401,-28.925442435894265 Z" fill="#00d8ca" stroke="#004b59" stroke-width="0.1" class="transition-fill duration-100 delay-75" data-testid="inactive-6"></path><path d="M17.052803287081844,-10.449971294318976,18.126155740732997,-8.452365234813989 L40.783850416649244,-19.017821778331474,38.36880739593415,-23.512435412217695 Z" fill="#00d8ca" stroke="#004b59" stroke-width="0.1" class="transition-fill duration-100 delay-75" data-testid="inactive-7"></path><path d="M18.410097069048806,-7.814622569785476,19.17639469736386,-5.680306894078453 L43.14688806906869,-12.78069051167652,41.42271840535982,-17.58290078201732 Z" fill="#00d8ca" stroke="#004b59" stroke-width="0.1" class="transition-fill duration-100 delay-75" data-testid="inactive-8"></path><path d="M19.362952807562156,-5.007600081088829,19.805361374831406,-2.7834620192013086 L44.56206309337067,-6.262789543202945,43.56664381701485,-11.267100182449864 Z" fill="#00d8ca" stroke="#004b59" stroke-width="0.1" class="transition-fill duration-100 delay-75" data-testid="inactive-9"></path><path d="M19.890437907365467,-2.0905692653530696,19.999238461283426,0.1745307099674787 L44.99828653788771,0.39269409742682704,44.7534852915723,-4.703780847044406 Z" fill="#00d8ca" stroke="#004b59" stroke-width="0.1" class="transition-fill duration-100 delay-75" data-testid="inactive-10"></path><path d="M19.48144032084623,0.850578053624052,19.259922641605186,3.050472068284502 L45.43366366737634,7.19598539185062,45.95621819276546,2.006491818805456 Z" fill="#00d8ca" stroke="#004b59" stroke-width="0.1" class="transition-fill duration-100 delay-75" data-testid="inactive-11"></path><path d="M-1.5688033694578483,-17.93150456565142,1.5688033694578465,-17.93150456565142 L4.0963199091399325,-46.82115081031204,-4.096319909139937,-46.82115081031204 Z" fill="#004b59" stroke="#004b59" stroke-width="0.1" class="transition-fill duration-100 delay-75" data-testid="active-0"></path><path d="M-2.6105238444010315,-19.828897227476208,-4.8384379119933545,-19.40591452551993 L-10.886485301985047,-43.66330768241984,-5.87367864990232,-44.61501876182147 Z" fill="#00d8ca" stroke="#004b59" stroke-width="0.1" class="transition-fill duration-100 delay-75" data-testid="inactive--1"></path><path d="M-5.512747116339984,-19.225233918766378,-7.653668647301798,-18.477590650225736 L-17.220754456429045,-41.5745789630079,-12.403681011764965,-43.25677631722435 Z" fill="#00d8ca" stroke="#004b59" stroke-width="0.1" class="transition-fill duration-100 delay-75" data-testid="inactive--2"></path><path d="M-8.293864853124784,-18.19922541753086,-10.300761498201087,-17.143346014042244 L-23.17671337095245,-38.57252853159505,-18.661195919530762,-40.94825718944444 Z" fill="#00d8ca" stroke="#004b59" stroke-width="0.1" class="transition-fill duration-100 delay-75" data-testid="inactive--3"></path><path d="M-10.892780700300547,-16.77341135890848,-12.721564405555277,-15.432491667754402 L-28.623519912499372,-34.72310625244741,-24.50875657567623,-37.74017555754408 Z" fill="#00d8ca" stroke="#004b59" stroke-width="0.1" class="transition-fill duration-100 delay-75" data-testid="inactive--4"></path><path d="M-13.252400964314749,-14.979114415780044,-14.862896509547884,-13.382612127177165 L-33.44151714648274,-30.11087728614862,-29.817902169708184,-33.7030074355051 Z" fill="#00d8ca" stroke="#004b59" stroke-width="0.1" class="transition-fill duration-100 delay-75" data-testid="inactive--5"></path><path d="M-15.32088886237956,-12.855752193730787,-16.677716441343364,-11.038739706241163 L-37.52486199302257,-24.837164339042616,-34.47199994035401,-28.925442435894272 Z" fill="#00d8ca" stroke="#004b59" stroke-width="0.1" class="transition-fill duration-100 delay-75" data-testid="inactive--6"></path><path d="M-17.052803287081847,-10.449971294318976,-18.126155740733,-8.452365234813987 L-40.78385041664925,-19.01782177833147,-38.368807395934155,-23.512435412217695 Z" fill="#00d8ca" stroke="#004b59" stroke-width="0.1" class="transition-fill duration-100 delay-75" data-testid="inactive--7"></path><path d="M-18.410097069048806,-7.814622569785476,-19.17639469736386,-5.680306894078453 L-43.14688806906869,-12.78069051167652,-41.42271840535981,-17.58290078201732 Z" fill="#00d8ca" stroke="#004b59" stroke-width="0.1" class="transition-fill duration-100 delay-75" data-testid="inactive--8"></path><path d="M-19.362952807562156,-5.007600081088828,-19.805361374831406,-2.7834620192013073 L-44.56206309337067,-6.262789543202941,-43.56664381701485,-11.267100182449862 Z" fill="#00d8ca" stroke="#004b59" stroke-width="0.1" class="transition-fill duration-100 delay-75" data-testid="inactive--9"></path><path d="M-19.890437907365467,-2.090569265353071,-19.999238461283426,0.17453070996747794 L-44.99828653788771,0.3926940974268253,-44.7534852915723,-4.70378084704441 Z" fill="#00d8ca" stroke="#004b59" stroke-width="0.1" class="transition-fill duration-100 delay-75" data-testid="inactive--10"></path><path d="M-19.48144032084623,0.8505780536240521,-19.259922641605186,3.050472068284503 L-45.43366366737634,7.1959853918506225,-45.95621819276546,2.0064918188054564 Z" fill="#00d8ca" stroke="#004b59" stroke-width="0.1" class="transition-fill duration-100 delay-75" data-testid="inactive--11"></path><text x="0" y="4" text-anchor="middle" alignment-baseline="middle" class="text-4xl">E</text><text x="0" y="17" text-anchor="middle" alignment-baseline="middle" class="text-[4px]">82.41</text></svg></div>"`;

exports[`E tuning - three over - snapshot 1`] = `"<div class="flex flex-col items-center"><svg width="100" height="100" class="w-100 h-70 max-w-full" viewBox="-50 -50 100 70"><path d="M2.610523844401034,-19.828897227476208,4.838437911993354,-19.40591452551993 L10.886485301985045,-43.66330768241984,5.8736786499023275,-44.61501876182147 Z" fill="#00d8ca" stroke="#004b59" stroke-width="0.1" class="transition-fill duration-100 delay-75" data-testid="inactive-1"></path><path d="M5.512747116339983,-19.225233918766378,7.653668647301797,-18.477590650225736 L17.22075445642904,-41.5745789630079,12.403681011764963,-43.25677631722435 Z" fill="#004b59" stroke="#004b59" stroke-width="0.1" class="transition-fill duration-100 delay-75" data-testid="active-2"></path><path d="M8.293864853124782,-18.199225417530865,10.300761498201084,-17.143346014042248 L23.176713370952438,-38.572528531595054,18.661195919530762,-40.94825718944445 Z" fill="#00d8ca" stroke="#004b59" stroke-width="0.1" class="transition-fill duration-100 delay-75" data-testid="inactive-3"></path><path d="M10.892780700300541,-16.773411358908483,12.721564405555279,-15.4324916677544 L28.62351991249938,-34.7231062524474,24.50875657567622,-37.740175557544084 Z" fill="#00d8ca" stroke="#004b59" stroke-width="0.1" class="transition-fill duration-100 delay-75" data-testid="inactive-4"></path><path d="M13.25240096431475,-14.979114415780042,14.862896509547884,-13.382612127177165 L33.44151714648274,-30.11087728614862,29.817902169708187,-33.70300743550509 Z" fill="#00d8ca" stroke="#004b59" stroke-width="0.1" class="transition-fill duration-100 delay-75" data-testid="inactive-5"></path><path d="M15.32088886237956,-12.855752193730785,16.677716441343364,-11.038739706241163 L37.52486199302257,-24.837164339042616,34.47199994035401,-28.925442435894265 Z" fill="#00d8ca" stroke="#004b59" stroke-width="0.1" class="transition-fill duration-100 delay-75" data-testid="inactive-6"></path><path d="M17.052803287081844,-10.449971294318976,18.126155740732997,-8.452365234813989 L40.783850416649244,-19.017821778331474,38.36880739593415,-23.512435412217695 Z" fill="#00d8ca" stroke="#004b59" stroke-width="0.1" class="transition-fill duration-100 delay-75" data-testid="inactive-7"></path><path d="M18.410097069048806,-7.814622569785476,19.17639469736386,-5.680306894078453 L43.14688806906869,-12.78069051167652,41.42271840535982,-17.58290078201732 Z" fill="#00d8ca" stroke="#004b59" stroke-width="0.1" class="transition-fill duration-100 delay-75" data-testid="inactive-8"></path><path d="M19.362952807562156,-5.007600081088829,19.805361374831406,-2.7834620192013086 L44.56206309337067,-6.262789543202945,43.56664381701485,-11.267100182449864 Z" fill="#00d8ca" stroke="#004b59" stroke-width="0.1" class="transition-fill duration-100 delay-75" data-testid="inactive-9"></path><path d="M19.890437907365467,-2.0905692653530696,19.999238461283426,0.1745307099674787 L44.99828653788771,0.39269409742682704,44.7534852915723,-4.703780847044406 Z" fill="#00d8ca" stroke="#004b59" stroke-width="0.1" class="transition-fill duration-100 delay-75" data-testid="inactive-10"></path><path d="M19.48144032084623,0.850578053624052,19.259922641605186,3.050472068284502 L45.43366366737634,7.19598539185062,45.95621819276546,2.006491818805456 Z" fill="#00d8ca" stroke="#004b59" stroke-width="0.1" class="transition-fill duration-100 delay-75" data-testid="inactive-11"></path><path d="M-1.5688033694578483,-17.93150456565142,1.5688033694578465,-17.93150456565142 L4.0963199091399325,-46.82115081031204,-4.096319909139937,-46.82115081031204 Z" fill="#00d8ca" stroke="#004b59" stroke-width="0.1" class="transition-fill duration-100 delay-75" data-testid="inactive-0"></path><path d="M-2.6105238444010315,-19.828897227476208,-4.8384379119933545,-19.40591452551993 L-10.886485301985047,-43.66330768241984,-5.87367864990232,-44.61501876182147 Z" fill="#00d8ca" stroke="#004b59" stroke-width="0.1" class="transition-fill duration-100 delay-75" data-testid="inactive--1"></path><path d="M-5.512747116339984,-19.225233918766378,-7.653668647301798,-18.477590650225736 L-17.220754456429045,-41.5745789630079,-12.403681011764965,-43.25677631722435 Z" fill="#00d8ca" stroke="#004b59" stroke-width="0.1" class="transition-fill duration-100 delay-75" data-testid="inactive--2"></path><path d="M-8.293864853124784,-18.19922541753086,-10.300761498201087,-17.143346014042244 L-23.17671337095245,-38.57252853159505,-18.661195919530762,-40.94825718944444 Z" fill="#00d8ca" stroke="#004b59" stroke-width="0.1" class="transition-fill duration-100 delay-75" data-testid="inactive--3"></path><path d="M-10.892780700300547,-16.77341135890848,-12.721564405555277,-15.432491667754402 L-28.623519912499372,-34.72310625244741,-24.50875657567623,-37.74017555754408 Z" fill="#00d8ca" stroke="#004b59" stroke-width="0.1" class="transition-fill duration-100 delay-75" data-testid="inactive--4"></path><path d="M-13.252400964314749,-14.979114415780044,-14.862896509547884,-13.382612127177165 L-33.44151714648274,-30.11087728614862,-29.817902169708184,-33.7030074355051 Z" fill="#00d8ca" stroke="#004b59" stroke-width="0.1" class="transition-fill duration-100 delay-75" data-testid="inactive--5"></path><path d="M-15.32088886237956,-12.855752193730787,-16.677716441343364,-11.038739706241163 L-37.52486199302257,-24.837164339042616,-34.47199994035401,-28.925442435894272 Z" fill="#00d8ca" stroke="#004b59" stroke-width="0.1" class="transition-fill duration-100 delay-75" data-testid="inactive--6"></path><path d="M-17.052803287081847,-10.449971294318976,-18.126155740733,-8.452365234813987 L-40.78385041664925,-19.01782177833147,-38.368807395934155,-23.512435412217695 Z" fill="#00d8ca" stroke="#004b59" stroke-width="0.1" class="transition-fill duration-100 delay-75" data-testid="inactive--7"></path><path d="M-18.410097069048806,-7.814622569785476,-19.17639469736386,-5.680306894078453 L-43.14688806906869,-12.78069051167652,-41.42271840535981,-17.58290078201732 Z" fill="#00d8ca" stroke="#004b59" stroke-width="0.1" class="transition-fill duration-100 delay-75" data-testid="inactive--8"></path><path d="M-19.362952807562156,-5.007600081088828,-19.805361374831406,-2.7834620192013073 L-44.56206309337067,-6.262789543202941,-43.56664381701485,-11.267100182449862 Z" fill="#00d8ca" stroke="#004b59" stroke-width="0.1" class="transition-fill duration-100 delay-75" data-testid="inactive--9"></path><path d="M-19.890437907365467,-2.090569265353071,-19.999238461283426,0.17453070996747794 L-44.99828653788771,0.3926940974268253,-44.7534852915723,-4.70378084704441 Z" fill="#00d8ca" stroke="#004b59" stroke-width="0.1" class="transition-fill duration-100 delay-75" data-testid="inactive--10"></path><path d="M-19.48144032084623,0.8505780536240521,-19.259922641605186,3.050472068284503 L-45.43366366737634,7.1959853918506225,-45.95621819276546,2.0064918188054564 Z" fill="#00d8ca" stroke="#004b59" stroke-width="0.1" class="transition-fill duration-100 delay-75" data-testid="inactive--11"></path><text x="0" y="4" text-anchor="middle" alignment-baseline="middle" class="text-4xl">E</text><text x="0" y="17" text-anchor="middle" alignment-baseline="middle" class="text-[4px]">82.41</text></svg></div>"`;
